Flow.Launcher.Infrastructure/PinyinAlphabet.cs

```diff
@@ -36,8 +36,8 @@ public string Translate(string content)
                 {
                     if (WordsHelper.HasChinese(content))
                     {
-                        var result = WordsHelper.GetPinyin(content, ";");
-                        result = GetFirstPinyinChar(result) + result.Replace(";", "");
+                        var result = WordsHelper.GetPinyin(content, " ");
```

Comment:
with this change, when i pinyin search 你好 for nihao it doesnt hit as it return splitted up 'ni hao'

Reply:
I don't think anybody would use Chinese to search for pinyin like characters.... If both are chinese, it will be fine because the string to compare will also contain space.

Reply:
I don't think anybody would use Chinese to search for pinyin like characters.... If both are chinese, it will be fine because the string to compare will also contain space.

Reply:
ok so we dont need to worry about chinese to pinyin.

Reply:
No, because the reason using pinyin is to make searching faster by reducing the process of typing Chinese, since typing Chinese require extra typing.